### DISCUSSION TITLE
Has panel--inverse been implemented? The style isn't appearing on either Beta or Live.

### DISCUSSION THREAD

**Discussion by TrishB70** [nhsevidence/NICE-Experience#160] [General]

**Comment by wa-rren-dev** [comment]
So there's a `panel--impact` which is similar? We possibly have a slight deviation in our naming which we can address.

If you go to https://design-system.nice.org.uk/storybook/?path=/story/panel--try-it-out and click on "knobs" (I know) then you can test the three current variants via the select list.

**Comment by wa-rren-dev** [comment]
![image](https://user-images.githubusercontent.com/6341474/109792667-a9315880-7c0b-11eb-93ed-86dce04441da.png)

**Comment by TrishB70** [comment]
Impact is perfect. Actually better.

**Comment by wa-rren-dev** [comment]
Also I notice it's not the standard teal background, it's one of the other ones. _I_ think we should have a standard `inverse` style as well so let me know if you agree and I'll put it in the backlog.

**Comment by TrishB70** [comment]
A standard inverse would be useful. And the background of --impact should ideally be the same as for the Hero container: #18646e? It looks odd currently (#517489).

**Comment by ediblecode** [comment]
Possibly v1 removed `--inverse` which might be a regression. We did do some rationalisation on panels to get the 3:

- default (grey, used for supporting info)
- impact which is teal(ish) background
- and primary which is white background and teal(ish) thick border

Looks like `--inverse` was in v0 (see https://nice-digital.github.io/nice-design-system/components/panel/#default-light-panel) but not sure what happened to it. Also not sure its use-case over the other panels: maybe it's just for asthetic purposes? If we can articulate a clear use-case for an inverse then I'm all for it as a 4th panel type. Hope that helps?

**Comment by ediblecode** [comment]
> the background of --impact should ideally be the same as for the Hero container:

I think it's actually _deliberately_ different to avoid teal overload!

**Comment by TrishB70** [comment]
Yes there was mention of Teal overload! But having an off-teal colour doesn't really work either. To my eye it looks odd. The two teals are clashing. That's the designer in me. --inverse was a very dark grey which worked.

**Comment by ediblecode** [comment]
> But having an off-teal colour doesn't really work either. To my eye it looks odd. The two teals are clashing.

sounds like that's something to look at too then.

> But `--inverse` was a very dark grey which worked

cool let's re-add this in then

**Comment by wa-rren-dev** [comment]
@TrishB70 @ediblecode Just moved this to a discussion.

If we decide what we're doing I'll create an issue for it.

Are we going to:

- Create `--inverse` variant as a dark gray
- Re-colour impact to the main teal? Or do we know that `--impact` is out there somewhere that might have an undesirable effect?

**Reply by ediblecode** [reply]
> Create --inverse variant as a dark gray

Yeah, I think so.

> Re-colour impact to the main teal?

Not sure. I'd favour keeping it as-is TBH, or explore one of the other blue-shades from the brand palette. It's probably an issue on pages with a big teal block like the homepage or find guidance. And if we have other panel types we can use there it might not be so much of an issue. What do you think?

**Comment by wa-rren-dev** [comment]
Created an issue for this here https://github.com/nice-digital/nice-design-system/issues/161

**Reply by gaynorc** [reply]
We'd agreed some standard panel styles and use cases towards the end of 2019. @ediblecode - do you remember workshopping it in the bat cave? It would be good to use those as we'd all collaborated on working out what we'd needed.

**Reply by ediblecode** [reply]
We did. Those are the 3 that are documented at https://design-system.nice.org.uk/storybook/?path=/story/panel--default-supporting:
- default panel for supporting info
- impact (blue3) for main call to action
- primary to group or separate content
Those are all still valid I think, the question is whether we need extra styles?

**Reply by gaynorc** [reply]
Ah, I didn't realise that there's a difference between the design system pages and storybook. That's my oversight.
I think we need to talk about this within our team. We need to work out if the panels we'd agreed don't meet our needs, or if we're changing that based on an edge case. Either way I think we need to get consensus as a group on what we need.
Can we hold off on this for a few days while I get everyone together.

**Reply by ediblecode** [reply]
Yes, makes sense - if you're happy to have that conversation

**Reply by ediblecode** [reply]
And @gaynorc sorry we need to update the docs for https://design-system.nice.org.uk/components/panel/ - it's missing those new panel types

**Reply by wa-rren-dev** [reply]
I'm going to lock this discussion, two new variants are going through code review now, examples of them are here #161 . @gaynorc @TrishB70 Any new variants you'd like, just open a new discussion!
